modules/gltf/editor/editor_scene_exporter_gltf_plugin.cpp

```diff
@@ -85,12 +80,13 @@ void SceneExporterGLTFPlugin::_popup_gltf_export_dialog() {
 		filename = root->get_name();
 	}
 	_file_dialog->set_current_file(filename + String(".gltf"));
+
 	// Generate and refresh the export settings.
 	_export_settings->generate_property_list(_gltf_document, root);
-	_settings_inspector->edit(nullptr);
-	_settings_inspector->edit(_export_settings.ptr());
+	_settings_inspector->update_tree();
+
 	// Show the file dialog.
-	_file_dialog->popup_centered_ratio();
+	_file_dialog->popup_file_dialog();
```

Comment:
This makes the dialog appear smaller, not sure if it's a problem. We use this method everywhere.